config/jobs/kubernetes/sig-testing/kubernetes-kind-presubmits.yaml

```diff
@@ -0,0 +1,268 @@
+presubmits:
+  kubernetes/kubernetes:
+  - name: pull-kubernetes-e2e-kind
+    optional: true
+    always_run: false
+    decorate: true
+    branches:
+    - release-1.14
+    labels:
+      preset-bazel-scratch-dir: "true"
+      preset-bazel-remote-cache-enabled: "true"
+      preset-dind-enabled: "true"
+    decoration_config:
+      timeout: 40m
+    path_alias: k8s.io/kubernetes
+    extra_refs:
+    - org: kubernetes
+      repo: kubeadm
```

Comment:
do we still need kubeadm references?

Reply:
i'm leaving the old branches alone for now